Modules/Sources/BuildSettingsKit/BuildSecrets.swift

```diff
@@ -0,0 +1,100 @@
+public struct BuildSecrets: Sendable {
+
+    public struct OAuth: Sendable {
```

Comment:
This is the WordPress.com app secret, right? What do you think about naming this to be dotcom specific, instead of a generic term "OAuth"?